R/appenders.R

```diff
@@ -232,6 +232,36 @@ appender_pushbullet <- function(...) {
 }
 
 
+#' Send log messages to ntfy
+#' @param title notification title
+#' @param tags emoji (or general tag) for notification. See [ntfy::emoji]
+#' @param ... parameters passed to [ntfy::ntfy_send], such as `title`
+#'     or `tags`. 
```

Comment:
`title` and `tags` cannot be passed here as those are explicit params defined previously, so maybe drop that reference here?


```suggestion
#' @param ... extra parameters passed to [ntfy::ntfy_send]
```